app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,247 +11,172 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
-$sidebar-small-width: 75%;
-$sidebar-medium-width: 65%;
-$sidebar-large-width: 45%;
-$sidebar-footer-height: 5em;
-
-.darkswarm {
-  .shop-filters-sidebar {
-    display: flex;
-    flex-direction: column;
-    height: 100%;
-
-    .background {
-      position: fixed;
-      top: 0;
-      right: 0;
-      z-index: 200;
-      height: 100%;
-      width: 100%;
-      background-color: $shop-sidebar-overlay;
-      opacity: 0;
-      transition: opacity $transition-sidebar;
-    }
+.shop-filters-sidebar {
+  .property-selectors {
+    margin-bottom: $sidebar-footer-height + 2em;
+  }
+}
 
-    &.shown {
-      .background {
-        opacity: 1;
-      }
+products {
+  display: block;
 
-      .sidebar, .sidebar-footer {
-        margin-right: 0;
-      }
+  @include breakpoint(tablet) {
+    input.button.right {
```

Comment:
ufff, this is going to explode soon. A right thing left aligned :exploding_head:

Reply:
I've changed the indentation here, but I'm not touching the element this rule applies to. I'd prefer to leave it alone for this PR.